roles/debian-hardening/tasks/main.yml

```diff
@@ -178,6 +181,14 @@
     state: absent
   when: revert
 
+- name: Wait for DHCP for SSH
```

Comment:
Wait for IP set
Because you can have a static IP (this is the case in the RTE CI)

Reply:
Commit moved out of this PR.